ODIN_II/exec_wrapper.sh

```diff
@@ -175,12 +175,13 @@ function display() {
 	esac
 
 	# check for uncaught errors
-
 	ERROR_CATCH="$(cat "${LOG_FILE}" | grep 'Odin exited with code: ' | awk '{print $5}' | grep -E '^\-?[0-9]+$')"
 	[ "_${ERROR_CATCH}" != "_" ] && EXIT_CODE="${ERROR_CATCH}"
 
 	EXIT_ERROR_TYPE=$( print_exit_type "${EXIT_CODE}" )
 
+	MISMATCH_CHECK="$(cat "${LOG_FILE}" | grep "Error::OUTPUT_BLIF Vector files differ.")"
```

Comment:
can you put that as a separate PR? this might some effect on the json and blow up the number of file changes otherwise. I'd rather see this self contained

Reply:
https://github.com/verilog-to-routing/vtr-verilog-to-routing/pull/1399